assets/js/src/form_editor/blocks/submit/edit.jsx

```diff
@@ -31,7 +31,7 @@ const SubmitEdit = ({ attributes, setAttributes }) => {
 
   const styles = !attributes.styles.inheritFromTheme ? {
     fontWeight: attributes.styles.bold ? 'bold' : 'inherit',
-    borderRadius: attributes.styles.borderRadius ? `${attributes.styles.borderRadius}px` : 0,
```

Comment:
Shouldn't we make it consistent the other way around? I mean we should probably allow a user to set 0 border-radius.